internal/distro/distro.go

```diff
@@ -36,23 +36,23 @@ var (
 	oemLookasideDir = "/usr/share/oem"
 
 	// Helper programs
-	chrootCmd     = "/usr/bin/chroot"
-	groupaddCmd   = "/usr/sbin/groupadd"
-	idCmd         = "/usr/bin/id"
-	mdadmCmd      = "/usr/sbin/mdadm"
-	mountCmd      = "/usr/bin/mount"
-	sgdiskCmd     = "/usr/sbin/sgdisk"
-	udevadmCmd    = "/usr/bin/udevadm"
-	usermodCmd    = "/usr/sbin/usermod"
-	useraddCmd    = "/usr/sbin/useradd"
-	restoreconCmd = "/usr/sbin/restorecon"
```

Comment:
this one actually get embedded in a systemd unit, which does need an absolute path